src/btree/bt_vrfy.c

```diff
@@ -434,7 +434,8 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
             memset(&addr_unpack, 0, sizeof(addr_unpack));
             addr_unpack.newest_start_durable_ts = ckpt->start_durable_ts;
             addr_unpack.oldest_start_ts = ckpt->oldest_start_ts;
-            addr_unpack.oldest_start_txn = ckpt->oldest_start_txn;
+            addr_unpack.oldest_start_txn =
+              ckpt->write_gen > S2C(session)->base_write_gen ? ckpt->oldest_start_txn : 0;
             addr_unpack.newest_stop_durable_ts = ckpt->stop_durable_ts;
             addr_unpack.newest_stop_ts = ckpt->newest_stop_ts;
             addr_unpack.newest_stop_txn = ckpt->newest_stop_txn;
```

Comment:
Don't you need to do the same there here (instead of 0 you'd want `WT_TXN_MAX` as default I think)? Probably @luke-pearson should confirm since he understands this issue better but I'd imagine you'd want to replicate the same rules as in `__wt_cell_unpack_dsk`.

Reply:
Agree with @tetsuo-cpp that it should follow the rules in `__wt_cell_unpack_dsk`.

Reply:
Done.